README.md

```diff
@@ -52,6 +54,30 @@ bosh update cloud-config ./example/cloud-config.yml
 	-d ./example/dbstubs/db-stub-external.yml \
 ```
 
+#### Deploy on bosh-lite with cf-deployment
+Install [Bosh-cli-v2](https://bosh.io/docs/cli-v2.html#install)
+
+Install and start [BOSH-Deployment](https://github.com/cloudfoundry/bosh-deployment), following its [README](https://github.com/cloudfoundry/bosh-deployment/blob/master/README.md). 
+
+Install [CF-deployment](https://github.com/cloudfoundry/cf-deployment/blob/master/cf-deployment.yml)
+
+Create and upload release
+```sh
+git clone https://github.com/cloudfoundry-incubator/app-autoscaler-release
+cd app-autoscaler-release
+./scripts/update
+bosh create-release
+bosh -e env upload-release
+```
+Deploy app-autoscaler
+```sh
+bosh -e env -d app-autoscaler \
+     deploy app-autoscaler-deployment.yml \
+     --vars-store=autoscaler-deployment-vars.yml \
```

Comment:
Yes, it gets generated during the deployment in the current directory and contains certs and passwords.